src/Core/Block.cpp

```diff
@@ -430,6 +433,16 @@ std::string Block::dumpIndex() const
     return out.str();
 }
 
+std::string Block::dumpContent() const
```

Comment:
We have `formatBlock` function. Calling it requires passing `output_format` but it can be easyly retieved, so the same would be 3 lines of code, a bit more, but I don't think it's worth intorudcing such a `dumpContent` methond.

https://github.com/bigo-sg/ClickHouse/blob/216dcbef2c86a58a0c8b3679395e89d3fec640b2/src/Formats/formatBlock.cpp#L11